src/contrib/testkits/Akka.TestKit.Xunit/Akka.TestKit.Xunit.csproj

```diff
@@ -18,5 +18,8 @@
   <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
     <DefineConstants>$(DefineConstants);RELEASE</DefineConstants>
   </PropertyGroup>
-
+  <PropertyGroup>
+    <!-- WORKAROUND: for some reason starting at Akka.NET 1.3.2 this package was determined as "unpackable" by default via DOTNET CLI -->
+    <IsPackable>true</IsPackable> 
```

Comment:
I have no idea what the .NET Core SDK suddenly started skipping these packages. Found out that's what was occurring when I ran `dotnet pack -v diag` on the individual Akka.TestKit.XUnit projects

Reply:
Yep, IsPackable is false by default for test projects, so you have to [opt-in](https://github.com/dotnet/templating/pull/377).